{
"apiVersion": "1.0",
"data":{
"error": true,
"errorMessage": "org.apache.http.conn.HttpHostConnectException: Connect to d100aeu91605.d100.intern:8901 [d100aeu91605.d100.intern/4.242.96.158] failed: A remote host did not respond within the timeout period."
},
"context": "137353393383500000000000",
"session": "Dialog-ID",
"service": "adf6c072-b5d7-4223-9d16-fefc2075c456",
"blz": 94059421,
"timestamp": "2016-11-15T13:29:39.312+0100"
}
